types/stylis/index.d.ts

```diff
@@ -6,20 +6,20 @@
 export as namespace stylis;
 
 export interface Element {
-    parent?: Element | undefined;
-    children?: Element[] | string | undefined;
-    root: Element;
+    parent: Element | null;
```

Comment:
Both `parent` and `root` originate here:
https://github.com/thysultan/stylis.js/blob/e6843c373ebcbbfade25ebcc23f540ed8508da0a/src/Parser.js#L25
and they start as nulls here:
https://github.com/thysultan/stylis.js/blob/e6843c373ebcbbfade25ebcc23f540ed8508da0a/src/Parser.js#L10